pxr/usdImaging/usdImaging/cameraAdapter.cpp

```diff
@@ -265,7 +265,7 @@ UsdImagingCameraAdapter::Get(UsdPrim const& prim,
         return vExposureResponsivity;
     } else if (key == HdCameraTokens->exposureScale) {
         // The computed linear exposure multiplier ratio
-        return VtValue(log2(cam.ComputeLinearExposureScale(time)));
+        return VtValue(cam.ComputeLinearExposureScale(time));
```

Comment:
This had been log2 throughout, to match the encoding of "exposure".  Is the loss of precision such that this is necessary for you all?  If so: (1) let's make the hydra token "linearExposureScale" and (2) let's update dataSourceCamera.cpp correspondingly.

Reply:
The thinking behind linear wasn't avoiding precision loss, so much as simplicity and consistency:

- the value is most easily expressed and calculated in linear space (since of the 5 component attributes, only 1 is in log2 space)
- the value will generally be used programmatically in linear space (ie, as a pixel value multiplier)
- as @anderslanglands points out, we formerly only converted to log2 where necessary for shoe-horning back into the exposure attribute.  Now that we're making "exposure" and "exposureScale" completely separate attributes, and not worrying about existing implementations picking up the new behavior / attributes "for free", we have the opportunity to make ALL usages of exposureScale in linear space.

Regardless, I totally agree with you that the possibility for confusion is high - so, in addition to making a commit where the token names are changed to "linearExposureScale", I also added a commit where ALL references to "exposureScale" in the code are changed to "linearExposureScale", for clarity.  (You can leave that commit out if you feel it's too verbose, though!)